src/contexts/FastUnstake/index.tsx

```diff
@@ -132,7 +132,7 @@ export const FastUnstakeProvider = ({
         // if localMeta existed, start checking from the next era.
         const eraTocheck = localMeta
           ? localMeta.checked[localMeta.checked.length - 1] - 1
-          : activeEra.index;
+          : activeEra.index.toNumber();
```

Comment:
This should ideally be BigNumber, as we are converting all era indexes to BigNumber.